kicker-portal-db/init-scripts/init.sql

```diff
@@ -1,3 +1,41 @@
 CREATE USER docker;
 CREATE DATABASE docker;
-GRANT ALL PRIVILEGES ON DATABASE docker TO docker;
\ No newline at end of file
+GRANT ALL PRIVILEGES ON DATABASE docker TO docker;
```

Comment:
You are using default db at the moment. Add following to switch context:
`USE docker;`

And lets rename "docker" to something more specific: kicker, kicker-db... etc